pkg/api/v1alpha1/iperf3_types.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2019 The xridge kubestone contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Iperf3Spec defines the Iperf3 Benchmark Job
+type Iperf3Spec struct {
```

Comment:
Just a note (lets talk about it later): I think it would be better to have a single "benchmark" CRD + controller then a separate for every benchmark (network, disk, cpu, etc.).

Reply:
I gravitate towards disagreing with this as there will be little commonality between network, disk and core k8s benchmarking. I'm open to re-visit this once we have more than one benchmark implemented.